data/build.sh

```diff
@@ -20,5 +20,6 @@ if [ -z "$plugin_name" ]; then
     exit 1
 fi
 
+cp -r $TYK_GW_PATH/vendor $PLUGIN_BUILD_PATH/vendor
```

Comment:
Golang plugin builds have issues with vendored dependencies (plugin won't load). There is some workorounds for that (i.e. copying all deps to GOPATH) and opened issue for Golang build